mmpose/models/backbones/shufflenet_v1.py

```diff
@@ -283,7 +283,7 @@ def make_layer(self, out_channels, num_blocks, first_block=False):
             out_channels (int): out_channels of the block.
             num_blocks (int): Number of blocks.
             first_block (bool, optional): Whether is the first ShuffleUnit of a
-                sequential ShuffleUnits. Default: False, which means not using
+                sequential ShuffleUnits. Default: False, which means using
```

Comment:
Default: True?

Reply:
I just keep the default value of `arg` and correct the docstring to avoid BC-breaking.